sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/AnalyticsTransmissionTarget.java

```diff
@@ -225,6 +232,15 @@ public void run() {
         return future;
     }
 
+    /**
+     * Getter for base context.
+     *
+     * @return the context.
+     */
+    Context getContext() {
```

Comment:
We are passing channel to PropertyConfigurator and not using a getter, please also pass Context to avoid this and make it consistent with channel field.

Reply:
[proposal.txt](https://github.com/Microsoft/AppCenter-SDK-Android/files/2362695/proposal.txt)

Reply:
That would also work indeed.